### ISSUE TITLE
 Alternative category layout does not work

### ISSUE THREAD

**Issue by caw67** [joomla/joomla-cms#30619]
Look here https://github.com/joomla/joomla-cms/issues/21800
how can i work with mutliple overrides???
all instructions says it have to work
i have choosen the new bloglayout in the category. but dont work

**Comment by chmst** [comment]
@caw67 Can you ask this question in the joomla forum (https://forum.joomla.org/)? There are supporters who will help you. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30619">issues.joomla.org/tracker/joomla-cms/30619</a>.</sub>

**Comment by caw67** [comment]
No, i think its a bug! I know how it works, but it dont work like joomla doc and other docs

**Comment by caw67** [comment]
I have choosen the layout in the category https://docs.joomla.org/J3.x:Layout_Overrides_in_Joomla but nothing works

**Comment by shim-sao** [comment]
It seems to works for articles in the category and sub-categories but not for the main category itself.

If you look at the ```tmpl/categories/default.php```, the layout of the category is hard code ```categories_default```.
_There is no way to change the template and layout._
Just an override of the layout is allowed.
We need to be allowed to assign a _tmpl_ like this ```tmpl/categories/want_we_want.php```

In the same time why image can be floated in the articles and not in the categories ? Any reason ?
For me there is missing some simple options for the display of the _categories_ in the _tmpl choice_ and the _default layout_.
Can I/we had them ?
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30619">issues.joomla.org/tracker/joomla-cms/30619</a>.</sub>
